torchvision/transforms/functional.py

```diff
@@ -577,3 +577,38 @@ def to_grayscale(img, num_output_channels=1):
         raise ValueError('num_output_channels should be either 1 or 3')
 
     return img
+
+
+def erase(img, x, y, w, h, value=0):
+    """ Erase the given Image with random values,
+        or the Imagenet mean pixel value.
+        'Random Erasing Data Augmentation' by Zhong et al.
+        See https://arxiv.org/pdf/1708.04896.pdf
+
+    Args:
+        img (Tensor): Image to be erased.
+        x: Upper pixel coordinate.
+        y: Left pixel coordinate.
+        w: Width of the erased image.
+        h: Height of the erased image.
+        values: Type of erasing value.
+                If values = 0, erasing with random values,
+                else, erasing with the Imagenet mean pixel value.
+
+    Returns:
+        Erased Image.
+    """
+    imagenet_mean_value = [0.485, 0.456, 0.406]
```

Comment:
I don't think we should hard-code the imagenet mean in the transform, but instead let the user pass it as an argument.